src/python/WMCore/Services/WorkQueue/WorkQueue.py

```diff
@@ -2,6 +2,7 @@
 from WMCore.Database.CMSCouch import CouchServer, CouchConflictError
 from WMCore.Lexicon import splitCouchServiceURL
 from WMCore.WMSpec.WMWorkload import WMWorkloadHelper
+from WMCore.WorkQueue.DataStructs.WorkQueueElement import STATES
```

Comment:
it looks like this import causes import problems when it's used by ReqMgr2. I'll have to add `WMCore.WorkQueue.DataStructs+` as a dependency for reqmgr2